controls/functions.js

```diff
@@ -1,23 +1,31 @@
-var express = require("express");
-var router = express.Router();
+var helper = {};
 var passport = require("passport");
 var bcrypt = require("bcryptjs");
-
-router.submitQuestion = async function (req, res, next) {
-
-    console.log(req.body.ques);
-    console.log(req.body.testcases);
-
-    /**
-     * Add the problem here.
-     * The qID of this problem will be the 
-     * largest qID from the collection plus 1
-     */
-
-    res.status(201).send('Success/ Failure');
+const Question = require("../models/problems"); 
+const TC = require("../models/testcases");
+
+helper.submitQuestion = async function (req, res, next) {
+
+    const ques = req.body.ques;
+    const tc = req.body.testcases;
+
+    try {
+        
+        const qID = 1 + await Question.estimatedDocumentCount();
```

Comment:
A better and time efficient approach can be to maintain another collection to store just the total number of problems created till now. Otherwise, when any problem is deleted afterward, then the qID of two problems can clash. You can create a model for it in the folder `./models/total_questions.js` to have just one field- `totalProb: Number`.
Increment this value everytime a question is successfully created, and this will be the qID of the current problem. This value will **not decrement** when any problem is deleted (if implemented in future).